### ISSUE TITLE
Keyword Searches

### ISSUE THREAD

**Issue by Jiros** [covidgraph/visual-graph-explorer#33]
I am curious about the behaviour of the publication and patent keyword search fields.

As an example - one of the Researcher use-case examples is to find all papers in the CORD-19 data that relate to kidney.

I would expect to be able to type "kidney" as a keyword and get the list of papers returned as search results. Instead the auto-populate on the field itself returns the list of papers and I can only load one at a time. The same behaviour is in effect on the Patent keyword search.

If this is the desired behaviour then I would suggest rewording the text on these fields to the effect of Find and load a publication/patent... and changing the button to 'Load Paper' and 'Load Patent' as only that one object will be added to the search result view.

**Comment by yGuy** [comment]
That should be working, right now, with the full-text search in place.
